content/src/scripts/containers/TreeList/styles.css

```diff
@@ -9,7 +9,7 @@
 .spantree-tree-list {
   scroll-behavior: smooth;
   overflow-y: auto;
-  height: calc(100vh - 40px);
+  height: calc(100vh - 90px);
```

Comment:
less space due to the 50px of the filter area.
Maybe there is a way to have -40 for repository view and -90 on merge request view, I wasnt able to do this without copy the entire css style and make it conditional on the pane...